### ISSUE TITLE
shm_ib/test_ucp_wireup_1sided.multi_ep_1sided/3 failure on boo31

### ISSUE THREAD

**Issue by dmitrygladkov** [openucx/ucx#3555]
```
21:19:52 [ RUN      ] shm_ib/test_ucp_wireup_1sided.multi_ep_1sided/3
21:19:54 [boo31:14828:0:14828] ucp_proxy_ep.c:214  Assertion `proxy_ep->uct_ep != ((void *)0)' failed
21:20:29 
21:20:29 /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucs/debug/assert.c: [ ucs_fatal_error_message() ]
21:20:29       ...
21:20:29        33     }
21:20:29        34 
21:20:29        35     ucs_handle_error(message_buf);
21:20:29 ==>    36     abort();
21:20:29        37 }
21:20:29        38 
21:20:29        39 void ucs_fatal_error_format(const char *file, unsigned line,
21:20:29 
21:21:03 ==== backtrace (tid:  14828) ====
21:21:03  0 0x000000000003bdc8 ucs_fatal_error_message()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucs/debug/assert.c:36
21:21:03  1 0x000000000003becf ucs_fatal_error_format()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucs/debug/assert.c:52
21:21:03  2 0x000000000001cc7d ucp_proxy_ep_replace()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucp/core/ucp_proxy_ep.c:214
21:21:03  3 0x000000000005ccd9 ucp_signaling_ep_am_short()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucp/wireup/signaling_ep.c:56
21:21:03  4 0x0000000000051c59 uct_ep_am_short()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/uct/api/uct.h:2127
21:21:03  5 0x0000000000051c59 ucp_tag_send_inline()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucp/tag/tag_send.c:159
21:21:03  6 0x0000000000051c59 ucp_tag_send_nb_inner()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucp/tag/tag_send.c:193
21:21:03  7 0x0000000000051c59 ucp_tag_send_nb()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../src/ucp/tag/tag_send.c:180
21:21:03  8 0x000000000069ac39 test_ucp_wireup::send_nb()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/ucp/test_ucp_wireup.cc:215
21:21:03  9 0x000000000069b541 test_ucp_wireup::send_recv()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/ucp/test_ucp_wireup.cc:326
21:21:03 10 0x000000000069c7d7 test_ucp_wireup_1sided_multi_ep_1sided_Test::test_body()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/ucp/test_ucp_wireup.cc:607
21:21:03 11 0x000000000050f2da ucs::test_base::run()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/test.cc:276
21:21:03 12 0x000000000050f31d ucs::test_base::TestBodyProxy()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/test.cc:302
21:21:03 13 0x000000000050512d HandleSehExceptionsInMethodIfSupported<testing::Test, void>()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3562
21:21:03 14 0x00000000004fb42c testing::Test::Run()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3635
21:21:03 15 0x00000000004fb6f8 testing::TestInfo::Run()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3812
21:21:03 16 0x00000000004fb8be testing::TestCase::Run()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3930
21:21:03 17 0x00000000004fb8be testing::TestCase::total_test_count()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3864
21:21:03 18 0x00000000004fb8be ???()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3929
21:21:03 19 0x00000000004fbea3 testing::internal::UnitTestImpl::RunAllTests()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:5802
21:21:03 20 0x00000000004fbea3 ???()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:5800
21:21:03 21 0x0000000000504cbd HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:3562
21:21:03 22 0x00000000004fa979 testing::UnitTest::Run()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:5416
21:21:03 23 0x00000000004fa979 ???()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/gtest-all.cc:5416
21:21:03 24 0x0000000000505705 main()  /scrap/jenkins/workspace/hpc-ucx-pr-3/label/boo31/worker/2/contrib/../test/gtest/common/main.cc:99
21:21:03 25 0x000000347981ed1d __libc_start_main()  ???:0
21:21:03 26 0x00000000004e16c9 _start()  ???:0
21:21:03 =================================
```
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/label=boo31,worker=2/10105/console

**Comment by dmitrygladkov** [comment]
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10400/label=boo31,worker=3/console

**Comment by dmitrygladkov** [comment]
```
20:38:33 [ RUN      ] shm_ib/test_ucp_wireup_1sided.multi_ep_1sided/2
20:38:36 [boo31:20695:0:20695] ucp_proxy_ep.c:214  Assertion `proxy_ep->uct_ep != ((void *)0)' failed
20:39:11 
20:39:11 /scrap/jenkins/workspace/hpc-ucx-pr/label/boo31/worker/1/contrib/../src/ucs/debug/assert.c: [ ucs_fatal_error_message() ]
20:39:11       ...
20:39:11        33     }
20:39:11        34 
20:39:11        35     ucs_handle_error(message_buf);
20:39:11 ==>    36     abort();
20:39:11        37 }
20:39:11        38 
20:39:11        39 void ucs_fatal_error_format(const char *file, unsigned line,
```
http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10415/label=boo31,worker=1/console

**Comment by dmitrygladkov** [comment]
@yosefe This can be closed as a duplicate for #3642 issue
checked with #3654 fix and can't reproduce

**Comment by yosefe** [comment]
duplicate of #3642
